packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1233,6 +1310,7 @@ class _Editable extends LeafRenderObjectWidget {
       ..obscureText = obscureText
       ..cursorWidth = cursorWidth
       ..cursorRadius = cursorRadius
+      ..cursorOffset = cursorOffset
       ..textSelectionDelegate = textSelectionDelegate;
```

Comment:
The constructor ones should all be in the update and make sure they have setters in the render object